### PR TITLE
AB#7367 - Speakers page adjustments, new pattern background

### PR DESCRIPTION
**Pull request by NellyKostadinova**

### CODE REVIEW

**Comment by jflheureux** [comment]
@NellyKostadinova, the build failed with a linting issue. Could you please look at it? Thanks

File: Website/src/rendering/src/assets/css/rest.css [hunk 1]
```diff
@@ -77,12 +77,11 @@
   content: '';
   width: 50%;
   height: 100%;
-  background:
-    -webkit-linear-gradient(
-      left,
-      rgba(255, 255, 255, 0) 0%,
-      rgba(255, 255, 255, 0.3) 100%
-    );
+  background: -webkit-linear-gradient(
```
Comment (jflheureux) [inline Website/src/rendering/src/assets/css/rest.css:80]: The build failed with a linting error. Could you please have a look at it? Thanks

File: Website/src/rendering/src/assets/css/layout/sections.css [hunk 2]
```diff
@@ -76,12 +76,38 @@
     @apply bg-white;
   }
 
+  &--dark-pattern,
+  &--light-pattern {
+    @apply relative;
+
+    &::before,
```
Comment (chpa7) [inline Website/src/rendering/src/assets/css/layout/sections.css:83]: There is a small gap on the left from the xl breakpoint and up.

image

Comment (chpa7) [reply Website/src/rendering/src/assets/css/layout/sections.css:83]: This is still there, only for the speakers section and from xl and up.

Comment (chpa7) [reply Website/src/rendering/src/assets/css/layout/sections.css:83]: This is still there, only for the speakers section and from xl and up.

**Review by chpa7 — APPROVED** [review]

**Pull request merged**